sycl/tools/sycl-xocc/bin/sycl_xocc.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python3
+"""sycl_xocc
+This is an extra layer of abstraction on top of the shell invocing the
+SDx compiler. As SPIR/LLVM-IR is a second class citizen in SDx for the
```

Comment:
Probably you can start cleaning the names like SDx, xocc, etc in another commit.